FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -820,8 +821,8 @@ - (void)signInAndRetrieveDataWithGameCenterCredential:(FIRGameCenterAuthCredenti
                                                }
                                                FIRAdditionalUserInfo *additionalUserInfo =
                                                    [[FIRAdditionalUserInfo alloc]
-                                                       initWithProviderID:
```

Comment:
A breaking change to consider when we move to Swift implementation. Global strings are problematic, especially across languages.